src/data/markdown/translated-guides/en/02 Using k6/05 k6 Options/02 Reference.md

```diff
@@ -421,10 +421,12 @@ $ k6 cloud --exit-on-running script.js
 ## Hosts
 
 An object with overrides to DNS resolution, similar to what you can do with `/etc/hosts` on
-Linux/Unix or `C:\\Windows\\System32\\drivers\\etc\\hosts` on Windows. For instance, you could set
+Linux/Unix or `C:\Windows\System32\drivers\etc\hosts` on Windows. For instance, you could set
 up an override which routes all requests for `test.k6.io` to `1.2.3.4`.
 
-You can also redirect only from certain ports or to certain ports.
+Starting from v0.28.0, you can also redirect only from certain ports or to certain ports.
+
+Starting from v0.42.0, you can use an asterisk (`*`) at the start of the host name, to avoid repetition. For example, `*.k6.io` would apply the override for all subdomains of `k6.io`.
```

Comment:
```suggestion
k6 also supports ways to narrow or widen the scope of your redirects:
- You can redirect only from or to certain ports.
- To save repetition, you can use an asterisk (`*`) as a wild card at the start of the host name.
For example, `*.k6.io` would apply the override for all subdomains of `k6.io`.
```

Reply:
I don't think we need to mention version numbers as it'll go stale. v0.28 is already ancient. I also tried to organize this information in some sort of coherent group. I added "wildcard" just to help `ctrl+f` or internet searchers.

Reply:
> I don't think we need to mention version numbers as it'll go stale. v0.28 is already ancient.

That's true, and I wondered about removing it as well, but I think it's important to mention when a new feature is added that it's only available since a specific version. Otherwise people will get confused when they try to use it, and it doesn't work.

I think we need better version management of the docs altogether. Am I wrong, or didn't we have a version dropdown users could select? In this case we could mark this feature as "New" for v0.42.0, and then remove the label in the next version.

Or we could have a strategy to update these "Starting from" labels, and be disciplined about removing them a few versions after they're introduced.

Anyway, I'm OK with your changes. I'll translate them to Spanish to be consistent.

Reply:
I just looked through our internal chat, and I saw that we recommend keeping a version reference for 6 releases / 1 year. I remember that I once removed a lot of old version references, but I forgot about the policy :facepalm: .

 > Otherwise people will get confused when they try to use it, and it doesn't work.
 
 This convinced me that it makes sense to put it in the new reference. I'm so used to seeing old, irrelevant references that I didn't think of how it would look from the other side.